### ISSUE TITLE
Matrix values issue

### ISSUE THREAD

**Issue by vfdev-5** [polyaxon/polyaxon#363]
### Describe the bug

Most probably there is an issue with values passed to `matrix` in yaml file for hyperparameter tuning. For example,
My config can accept `True` and `False` as string parameters. In polyaxonfile.yaml it is set like:

  ```matrix:
    enable_option:
      values: [True, False]
```

When group experiments is launched by CLI everything goes well, however no experiment is created/scheduled. No pods in the namespace neither.
If I fix the config as `"True"`, `"False"`, group is scheduled and all goes as expected.

**Comment by mouradmourafiq** [comment]
Can you provide more info on how are you using these values?
just using the a param `values: [True, False]` works, do you intend to use the values as boolean or strings?

**Comment by vfdev-5** [comment]
@mouradmourafiq actually, it used like this:
```
python main.py --params enable_option=True
```
which is then passed into `eval`. Actually, now I have doubt where was the problem, just at that time passing `True`/`False` into `"True"`/`"False"` solved the problem...

**Comment by mouradmourafiq** [comment]
I think the issue is in the python parser. I just pass the value: `python main.py --value={{value}}` coming from `values: [True, False]` and it works.

**Comment by vfdev-5** [comment]
I see my total command is :
```
- python -u code/main.py --use_tb_logger --params data_path="{{ data_path }}" \
        seed=17 num_epochs={{ num_epochs }} \
        batch_size=512 final_weight={{ final_weight }} \
        enable_mixup={{ enable_mixup }} mixup_proba={{ mixup_proba }} mixup_alpha={{ mixup_alpha }} \
        lr_param_group_0="[(0, {{ lr_param_group_0_i }}), ({{ lr_param_group_0_e }}, {{ lr_param_group_0_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_1="[(0, {{ lr_param_group_1_i }}), ({{ lr_param_group_1_e }}, {{ lr_param_group_1_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_2="[(0, {{ lr_param_group_2_i }}), ({{ lr_param_group_2_e }}, {{ lr_param_group_2_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_3="[(0, {{ lr_param_group_3_i }}), ({{ lr_param_group_3_e }}, {{ lr_param_group_3_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_4="[(0, {{ lr_param_group_4_i }}), ({{ lr_param_group_4_e }}, {{ lr_param_group_4_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_5="[(0, {{ lr_param_group_5_i }}), ({{ lr_param_group_5_e }}, {{ lr_param_group_5_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_6="[(0, {{ lr_param_group_6_i }}), ({{ lr_param_group_6_e }}, {{ lr_param_group_6_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_7="[(0, {{ lr_param_group_7_i }}), ({{ lr_param_group_7_e }}, {{ lr_param_group_7_v }}), ({{ num_epochs }}, 0.0)]" \
        lr_param_group_8="[(0, {{ lr_param_group_8_i }}), ({{ lr_param_group_8_e }}, {{ lr_param_group_8_v }}), ({{ num_epochs }}, 0.0)]"

```

**Comment by mouradmourafiq** [comment]
Not sure which one is breaking but in any case, in v0.5 the platform will have the possibility to set/force types in the specification.

**Comment by mouradmourafiq** [comment]
I will close this as I could not reproduce it, and you seem to have found a way around. Hopefully such issues could be resolved with the type option in v0.5
